yos/sort_list.go

```diff
@@ -0,0 +1,43 @@
+package yos
+
+type SortListByName []*FilePathInfo
+
+func (n SortListByName) Len() int {
+	return len(n)
+}
+
+func (n SortListByName) Less(i, j int) bool {
+	return n[i].Info.Name() < n[j].Info.Name()
+}
+
+func (n SortListByName) Swap(i, j int) {
+	n[i], n[j] = n[j], n[i]
+}
+
+type SortListBySize []*FilePathInfo
+
+func (s SortListBySize) Len() int {
+	return len(s)
+}
+
+func (s SortListBySize) Less(i, j int) bool {
+	return s[i].Info.Size() < s[j].Info.Size()
+}
+
+func (s SortListBySize) Swap(i, j int) {
+	s[i], s[j] = s[j], s[i]
+}
+
+type SortListByModTime []*FilePathInfo
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported type SortListByModTime should have comment or be unexported](https://app.codacy.com/manual/an9an63/gut/pullRequest?prid=4963210)